extensions/media-preview/media/imagePreview.js

```diff
@@ -356,9 +356,26 @@
 
 	async function copyImage() {
 		try {
-			await navigator.clipboard.write([new ClipboardItem({
-				'image/png': fetch(image.src).then(request => request.blob())
-			})]);
+			if (!document.hasFocus()) {
+				// copyImage is called at the same time as webview.reveal, which means this function is running whilst the webview is gaining focus.
+				// Since navigator.clipboard.write requires the document to be focused, we need to wait for focus.
+				// We cannot use a listener, as there is a high chance the focus is gained during the setup of the listener resulting in us missing it.
+				setTimeout(copyImage, 10);
```

Comment:
Let's limit the number of retries on this. If the document doesn't get focused for some reason, I don't want this to be looping forever

Reply:
I've added a basic mechanism to limit it to 3 retries

Reply:
Increased to 5 retries 20ms apart. From testing 3 at 10 was regularly not enough, but 3 at 20 seems fine. Increasing to 5 retries at 20ms to be safe.